app-localize-behavior.html

```diff
@@ -227,6 +232,10 @@
     };
   },
 
+  __languageChanged: function() {
+    this.fire('app-language-changed', this.language);
```

Comment:
Done. Feel free to improve the wording if you feel there is a better description.